builtin/v10/evm/evm_actor_state.go

```diff
@@ -0,0 +1,10 @@
+package evm
+
+import (
+	"github.com/ipfs/go-cid"
+)
+
+type State struct {
+	Bytecode      cid.Cid
+	ContractState cid.Cid
```

Comment:
Nonce?

Reply:
it doesn't have such a field in its state!

Reply:
oh wait, it does. Recent addition, will fix.

Reply:
Fixed in https://github.com/filecoin-project/go-state-types/pull/116.